plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/GitHook.kt

```diff
@@ -86,6 +95,7 @@ private fun KtlintPlugin.PluginHolder.addInstallGitHookFormatTask() {
         it.description = "Adds git hook to run ktlintFormat on changed files"
         it.group = HELP_GROUP
         it.taskName.set(FORMAT_PARENT_TASK_NAME)
+        it.shouldUpdateCommit.set(true)
```

Comment:
I think this is appropriate?
```suggestion
        it.shouldUpdateCommit.convention(true)
```

Reply:
I would say no. As I see from documentation - `.convention(value)` should be used to set default value of `Property`. In this line value is pretty specific to this task.

Reply:
Perhaps we should set the convention inside of the task?

Reply:
that will make sense :+1:

Reply:
@JLLeitschuh I've set convention on task property (default is `false`) and forced-pushed.

Reply:
Why is the convention false but this is set to `true` here?

Reply:
I've set convention in task class `KtlintInstallGitHookTask`. In this particular lines specific task "addKtlintFormatGitPreCommitHook" is registered and `shouldUpdateCommit` changed to `true` as hook will run `ktlint --format` over staged files.

Reply:
Can you leave that as a comment to explain that to the next person that sees this?

Reply:
Added a comment in updated commit https://github.com/JLLeitschuh/ktlint-gradle/pull/343/commits/43fabe7bbc8c649266a0b7c2c4a4a5ccd7fe4aa2